.config/shell/profile

```diff
@@ -173,3 +173,7 @@ export VALE_CONFIG_PATH="${HOME}/.config/vale/vale.ini"
 # Loads support for direnv to load .envrc defined in specific directory
 command -v direnv > /dev/null && eval "$(direnv hook zsh)"
 
+eval "$(pyenv init --path)"
```

Comment:
Initialising pyenv was causing performance issue with shell loading.
So we are using https://github.com/davidparsson/zsh-pyenv-lazy plugin in zshrc.

Reply:
Got your point.